roles/ncn-common-setup/tasks/gcp.yml

```diff
@@ -46,3 +62,13 @@
     name: "gcp authorized keys"
     minute: "*/1"
     job: "cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys >> /var/log/cray/cron.log 2>&1"
+
+- name: get available suse extensions
+  command: /usr/sbin/SUSEConnect --list-extensions
+  register: suse_extensions
+  changed_when: False
+
+- name: activate suse public cloud module
+  command: /usr/sbin/SUSEConnect -p {{ suse_extensions.stdout | regex_search ('sle-module-public-cloud.*') }}
```

Comment:
This should probably read from a list, so if wanted to add or remove we could update a config instead of the task.

Like added to defaults/main.yml